src/evict/evict_page.c

```diff
@@ -205,7 +205,7 @@ __wt_evict(WT_SESSION_IMPL *session, WT_REF *ref, uint8_t previous_state, uint32
     else
         WT_ERR(__evict_page_dirty_update(session, ref, flags));
 
-    if (LF_ISSET(WT_EVICT_CALL_URGENT)) {
+    if (time_start != 0 /* WT_EVICT_CALL_URGENT */) {
```

Comment:
This comment seems out of place, `WT_EVICT_CALL_URGENT` probably left by accident?

Reply:
I did that deliberately --  this is the one change where it wasn't just using `time_start != 0` to turn on timings, but it's using `time_start` as a test for other things, which felt ugly. Thoughts?

I could split stuff apart:```
if (WT_EVICT_CALL_URGENT) {
}
if (time_start) {
}
```
it's not like it's expensive... I'll do that for clarity.

Reply:
@luke-pearson, I tried this, and it's worse -- I'll just remove the comment as you suggested, it's confusing.

Reply:
Fixed in f395b6935.